R/rdeephaven/README.md

```diff
@@ -0,0 +1,70 @@
+# The Deephaven R Client
+
+The Deephaven R client is an R package that enables R users to interface with a Deephaven server and perform various
+server-side operations from the comfort of RStudio or any other R interface.
+
+## What can the R client do?
+
+The Deephaven Client currently provides three basic functionalities:
+
+1. Connecting to a Deephaven server
+   -   with anonymous authentication, without the requirement for a username or password
+   -   with basic authentication that requires a username and password
+   -   with custom authentication that requires general key-value credentials
+
+2. Interfacing with Deephaven Tables on the server
+   -   Retrieve references to tables on the server without importing any of their data locally
+   -   Pull table data into an [Arrow RecordBatchReader](https://arrow.apache.org/docs/r/reference/RecordBatchReader.html),
+an [Arrow Table](https://arrow.apache.org/docs/r/reference/Table.html),
+a [dplyr Tibble](https://tibble.tidyverse.org),
+or an [R Data Frame](https://stat.ethz.ch/R-manual/R-devel/library/base/html/data.frame.html)
+   -   Create new tables on the server from any of the formats listed above
+   -   Bind new server-side tables to variable names, enabling access from outside the current R session
+
+3. Running scripts on the server
+   -   If the server is equipped with a console, run a script in that console
+   -   Currently, Python and Groovy are supported
+
+## Installation
+
+Currently, the R client is only supported on Ubuntu 20.04 or 22.04 and must be built from source, since the build process
+makes heavy use of the build instructions provided [here](https://github.com/deephaven/deephaven-core/tree/main/cpp-client).
+
+1. Clone this repository into a local directory using git's sparse-checkout or subversion.
+2. From the new local copy of this repo, run the following commands:
+   ```
+   cd lib
+   chmod +x build-cpp.sh
+   ./build-cpp.sh
+   ```
+   This will build all of the C++ client dependencies in a manner that is compatible with Rcpp, and will then build the Deephaven C++ client.
+3. With the C++ client installed, start an R console and build and install the R client as follows:
+   ```
+   R
+   >> install.packages("path/to/rdeephaven", repo=NULL, type="source", dependencies=TRUE)
+   ```
```

Comment:
```suggestion
   ```bash
   R
   ```
   ```R
   install.packages("path/to/rdeephaven", repo=NULL, type="source", dependencies=TRUE)
   ```
```